annubes/task.py

```diff
@@ -280,7 +282,7 @@ def plot_trials(self, n_plots: int = 1) -> go.Figure:
                 col=1,
             )
             fig.add_vline(
-                x=self._iti[i] + self.fix_time + self.dt,
+                x=self._fix_time[i] + self.dt,
```

Comment:
where did `iti` go? I thought we wanted to keep it as an option, but default it to 0 for our experiments.
And it is still present in `_setup_trial_phases`

Reply:
Is it possible (and easy) to test that a plot looks the way it's expected? I didn't test it now, so could easily be missing it if it's truly not supposed to be in the plot.